sandstone/urls.py

```diff
@@ -2,8 +2,8 @@
 from sandstone.lib.handlers.logout import LogoutHandler
 from sandstone.lib.handlers.main import MainHandler
 from sandstone.lib.handlers.state import StateHandler
+from sandstone.lib.handlers.update_handler import UpdateHandler
```

Comment:
@saurabhsood91 For consistency with the rest of the handlers, rename the file from `update_handler` to `update`.